kb/2023-08-11/using_netapp_third_party_storage.md

```diff
@@ -160,15 +160,15 @@ The procedure is as follows.
       apiVersion: storage.k8s.io/v1
       kind: StorageClass
       metadata:
-      name: ontap-san-economy
+        name: ontap-san-economy
       provisioner: csi.trident.netapp.io
       parameters:
-      selector: "name=default_backend_san"
+        "selector": "name=default_backend_san"
```

Comment:
Is the double quote on the selector needed? rarely see a key quoted.

Reply:
Per test, it worked. seems no harm with doule quota to express this is a string not a defined struct field.

Reply:
I'd suggest dropping the quote if there are no special characters in the key.
Some examples on the official doc. https://kubernetes.io/docs/concepts/storage/storage-classes/

Reply:
updated.